### PR TITLE
Add base policies

### PR DESCRIPTION
**Pull request by jemassey**
* Add a computed metric to calculate the percentage of unhealthy hosts
* Add policy for Active Flows Count (connections)
* Add policy for Process Bytes
* Add two policies for Unhealthy hosts (warning at 50% and critical at 75%)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ For detailed information on this package, please refer to the [online documentat
 
 ## Release History
 
+### Version 1.1.0
```
Comment (TheConnMan) [inline README.md]: Please keep this under `Version next` until you are about to merge to `master`.

File: analyticConfigurations/computed-aws.nlb.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "netuitive.aws.networkelb.unhealthyhostpercent",
+        "properties": {
+          "expression": "(data['aws.networkelb.unhealthyhostcount'].actual + data['aws.networkelb.healthyhostcount'].actual) == 0 ? 0 : (data['aws.networkelb.unhealthyhostcount'].actual / (data['aws.networkelb.unhealthyhostcount'].actual + data['aws.networkelb.healthyhostcount'].actual))*100",
```
Comment (TheConnMan) [inline analyticConfigurations/computed-aws.nlb.json:7]: If I'm understanding this correctly the value will be 0% if there are no hosts at all?

**Review by TheConnMan — APPROVED** [review]
LGTM

File: analyticConfigurations/computed-aws.nlb.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+          "fqn": "netuitive.aws.networkelb.unhealthyhostpercent",
```
Comment (bspindler) [inline analyticConfigurations/computed-aws.nlb.json:8]: let's not prefix this with 'netuitive' just remove it, we've been slowly moving away from this approach and even calculating these locally where possible.  So the fqn would just be aws.networkelb.unhealthyhostpercent

File: policies/aws.nlb.high.activeflowcount.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      },
+      {
+        "analytic": "actual",
+        "level": 1000,
+        "metric": "aws.networkelb.activeflowcount",
+        "operator": ">="
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a Critical alert when we detect unusually high Active Flow Counts (connections) with a minimum of 1000 Active Flows for at least 15 minutes.",
```
Comment (bspindler) [inline policies/aws.nlb.high.activeflowcount.json:23]: I don't like this policy.  It suggests jumps in connections are only relevant for this with >1000 connections.  Is it really Critical if we get an abnormally high (baselineDeviation) amount of connections if everything else is fine?  Which we can't answer so at best this is Warning, probably Info.

File: policies/aws.nlb.high.processedbytes.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      },
+      {
+        "analytic": "actual",
+        "level": 1000000000,
+        "metric": "aws.networkelb.processedbytes",
+        "operator": ">="
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a Critical alert when we detect unusually high Processed Bytes (connections) with a minimum of 1Gb Processed for at least 15 minutes.",
```
Comment (bspindler) [inline policies/aws.nlb.high.processedbytes.json:23]: same as connections comment

**Pull request merged**